src/events.js

```diff
@@ -600,6 +605,22 @@ export const addEventCheckInCompleteForm = (isCheckedIn, checkOutFlag) => {
     });
 };
 
+/**
+ * Checks if the participant has a clinical blood or urine collected variable under baseline. If participant has clinical blood or urine collected, show a notification and return true.
+ * @param {Object} data - participant data
+ * @returns {Boolean} - true if participant has any clinical blood or urine collected, false otherwise
+*/
+const checkClinicalBloodOrUrineCollected = (data) => {
+    const isBloodOrUrineCollected = data?.[conceptIds.collectionDetails]?.[conceptIds.baseline.visitId]?.[conceptIds.clinicalBloodOrUrineCollected];
+
+    if (isBloodOrUrineCollected === conceptIds.yes) { 
+        const bodyMessage = 'Check In not allowed, participant already has clinical collection for this timepoint.'
```

Comment:
Suggesting a big adjustments on text: "Check in not allowed: participant already has clinical collection for this timepoint."

Reply:
The wording of this message is not finalized yet by the biospecimen team and will be subject to change.